Description: Bundler
 Remove the dependency on bundler
Author: Balasankar C <balasankarc@autistici.org>
Last-Update: 2014-12-24
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/test/test_helper.rb
+++ b/test/test_helper.rb
@@ -1,4 +1,3 @@
-require 'bundler/setup'
 require 'minitest/autorun'
 require 'mocha/setup'
 require 'rack'
